django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -90,26 +90,32 @@
     display: block;
     text-indent: -3000px;
     overflow: hidden;
+    cursor: default;
+    opacity: 0.3;
```

Comment:
This widget originally used color changes to indicate the button being enabled/disabled (using `.active` ...bad class name, might change that in another commit). This commit switches that indication to the more conventionally used opacity property.

The color change is now used for the :hover and :focus pseudo-classes, which I also believe is more conventional.